macos/FluentUI/Button/Button.swift

```diff
@@ -188,6 +188,7 @@ open class Button: NSButton {
 	}
 
 	open override func mouseUp(with event: NSEvent) {
+		_ = target?.perform(action, with: event)
```

Comment:
Cann you add a comment explaining why we specifically want to trigger the action here, as opposed to `mouseDown` or somewhere else? And why you have to assign to `_`

Reply:
Obsolete, but I added comments to the updated version.